src/test/java/org/reaktivity/specification/socks/internal/SocksFunctionsTest.java

```diff
@@ -81,7 +81,7 @@ public void shouldBuildRouteExWithDomainName() throws Exception
     }
 
     @Test
-    public void shouldBuildRouteExWithDomainName1() throws Exception
+    public void shouldBuildRouteExWithSimpleDomainName() throws Exception
```

Comment:
A "simple" domain name is one without `.`s such as `"example"`, which is what you had in the code when I provided the suggestion to rename the test.

Now you have renamed the method, but also changed the code so it no longer aligns with the description.

Did you mix up simple and mid-level domain name test names?

Reply:
yes